en/Obsidian Sync/Select files and settings to sync.md

```diff
@@ -0,0 +1,36 @@
+# Select files and settings to sync
+
+Any files or settings that have been synced to your [[Local and remote vaults|remote vault]] count towards your [[Limitations#How large can each remote vault be|storage space]]. By default, [[Introduction to Obsidian Sync|Obsidian Sync]] ignores larger files such as audio, video, and PDFs.
+
+**Notes:**
+
+- Synced files remain in your remote vault even if you exclude them later on. If possible, configure the files and settings you want to sync before you start syncing your vault.
+- Settings are only synced during start-up. If you change what settings to sync, you need to restart Obsidian on your other devices for the new changes to take effect.
+
+## Select settings to sync
+
+1. Open **Preferences** -> **Sync**.
+2. Under **Vault configuration sync**, enable the settings you want to sync.
+
+## Select file types to sync
+
+1. Open **Preferences** -> **Sync**.
+2. Under **Selective sync**, enable the file types you want to sync.
+
+## Exclude folder from being synced
+
+By default, Obsidian recursively syncs all folders in your vault. If you don't want Obsidian to sync a certain folder, you can exclude it.
```

Comment:
I think "all files and folders" better than "recursively syncs all folders"? I know "recursively" is more technically correct, but I don't know how widely known the word is. Judging from the context it should be pretty clear what it means imo.